.eslintrc.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+  extends: ["eslint:recommended"],
+  parserOptions: {
+    ecmaVersion: 6,
+  },
+  env: {
+    node: true,
+  },
+  rules: {
+    semi: ["error", "never"],
+    'no-console': "off",
+  },
+  globals: {
+    Promise: true,
```

Comment:
Instead of specifying `Promise` here I would just add `browser` to the list of environments, as that is more accurate.